website/docs/events/m1.md

```diff
@@ -40,7 +42,125 @@ Please consider topics such as:
+LB: Now, information is hard to access, private and usually one-way.
+LB: We want it publically available, both-way, we need a big community outside of university.
+LB: We currently don't have acquaintances in other countries, so we have to grow the visibility of this project.
+LB: Ideally, in the future, we would have a large number of repositories, wach having maintainers, contirbuters and users.
+LB: We have multiple resources - [Discord](https://bit.ly/OpenEduHub), [GitHub](https://github.com/Open-Education-Hub), [website](https://open-education-hub.github.io/), Google Groups, [Twitter](https://twitter.com/OpenEduHub), [Linkedin](https://www.linkedin.com/company/open-education-hub), [Facebook](https://www.facebook.com/openeducationhub), [Reddit](https://www.reddit.com/user/Open_Education_Hub), [Youtube](https://www.youtube.com/@OpenEduHub), Instagram.
+BB: You said that there should be a sort of guideline of what is acceptable for each repository, who is responsible of creating this?
+LB: The maintainers, they know the base idea of the repository and know what is accetable and what not.
+
+## Infrastructure
+
+SW: A problem in out faculty is that educators spend too much time with setup actions. These things do not help us educate student better. They only take time, so we have less time to spend on thinking how to educate others better.
+SW: The difference between universities and Coursera is that we have a mentoring component.
+SW: Our goal is to automate the setup actions as much as possible. The eduator creates a new instance for his course, he updates it every semester with a minimum effort and publishes the content into his desired format.
+SW: We want to create a simple method for educators to do so - for both technical and nontechnical people.
+SW: We want to reuse as many already existing solutions.
+SW: We absolutely need to avoid getting into a situationn similar to [VMChecker](https://vmchecker.cs.pub.ro/ui/#) that is manintained only by Răzvan. There is a single point of failure, any problem has to go to Răzvan and it's complicated.
+SW: Keeping the infrastructure up, generating homework, grading homework, checking quizes are all things that take time.
+SW: An ideal course is describes using a GitHub repository, it's open source, students can access materials easily, they can contibute and correct any problems they might find. We have an example of this model at the [Operating Systems course](https://github.com/open-education-hub/operating-systems).
+SW: The content is all on GitHub, in Markdown format.
+SW: Anybody can cretae a fork and update the content.
+SW: If this were to be on OCW, it would have been difficult for me to update anything there. I don't have write access to the [Operating Systems class on OCW](https://ocw.cs.pub.ro/courses/so). I can copy paste the text (without the wiki format), but I cannot see the source of the page.
```

Comment:
```suggestion
SW: If this were to be on OCW, it would be difficult for me to update anything there. I don't have write access to the [Operating Systems class on OCW](https://ocw.cs.pub.ro/courses/so). I can copy and paste the text (without the wiki format), but I cannot see the source of the page.
```